chart/values.schema.json

```diff
@@ -1608,21 +1689,65 @@
                                 }
                             ],
                             "$ref": "#/definitions/io.k8s.api.core.v1.ResourceRequirements"
+                        },
+                        "containerSecurityContext": {
+                            "description": "Container security context definition. The values in this parameter will be used when `containerSecurityContext` is not defined for specific Containers",
+                            "type": "object",
+                            "$ref": "#/definitions/io.k8s.api.core.v1.SecurityContext",
+                            "default": {
+                                "runAsUser": 0
+                            },
+                            "x-docsSection": "Kubernetes",
+                            "examples": [
+                                {
+                                    "allowPrivilegeEscalation": false,
+                                    "capabilities": {
+                                        "drop": [
+                                            "ALL"
+                                        ]
+                                    }
+                                }
+                            ]
                         }
                     }
                 },
-                "securityContext": {
```

Comment:
We need to leave these old deprecated ones in the schema also, otherwise helm won't allow you to actually deploy any longer.

Might be worth adding a test or 2 to cover the old ones to make sure they keep working.

Reply:
added some legacy tests and also readded the schema for the deprecated securityContext, lets see what the test say :)